dbt_project.yml

```diff
@@ -1,25 +1,25 @@
 #Welcome to your Spellbook!
 
-name: 'spellbook'
-version: '1.0.0'
+name: "spellbook"
```

Comment:
this is not a big deal but is your linter configurable? it would be nice not to have these quote switches in this PR. but don't fuss with it if it's annoying to turn off.